requirements.in

```diff
@@ -1,25 +1,25 @@
-pycparser==2.14 # TEMPORARY
+-e git+https://github.com/steve-gregory/billiard.git#egg=billiard # TEMPORARY
```

Comment:
Including this until a version of billiard has been released that includes https://github.com/celery/billiard/pull/216